modules/administration/pages/kubernetes.adoc

```diff
@@ -1,175 +1,173 @@
 [[kubernetes]]
 = Kubernetes
 
+You can use {productname} to manage {k8s} containerized clients in addition to regular traditional or Salt clients.
 
+{productname} manages {k8s} clusters with a virtual host manager.
+For more information on virtual host managers, see xref:client-configuration:virt-vmware.adoc[]
 
-[[at.k8s.integration.requires]]
-== Prerequisites
 
-The prerequisites listed below should be met before proceeding.
+To use {productname} with {k8s}, you will need to have your{productname} Server configured for container management, with all required channels present, and a registered container build host available.
 
-* At least one Kubernetes or SUSE CaaS Platform cluster available on your network
-* {productname} configured for container management
-(required channels are present, a registered container build host available, etc.)
-* [package]#virtual-host-gatherer-Kubernetes# package installed on your {productname} server
+You will also require:
 
-
-
-[[at.requirements]]
-== Requirements
-
-
-* Kubernetes version 1.5.0 or higher.
-Alternatively, you can use SUSE CaaS Platform
+* At least one {k8s} or {caasp} cluster available on your network.
+* The [systemitem]``virtual-host-gatherer-Kubernetes`` package installed on the {productname} Server.
+* {k8s} version 1.5.0 or higher, or {caasp}.
 * Docker version 1.12 or higher on the container build host
 
 
 
-[[at.registering.kubernetes.as.a.virtual.host.manager]]
-== Register Kubernetes as a Virtual Host Manager
+== Create a {k8s} Virtual Host Manager
 
+{k8s} clusters are registered with {productname} as a virtual host manager (VHM).
 
-_Kubernetes_ clusters are registered with SUSE Manager as ``virtual host managers``.
-Registration and authorization begins with importing a `kubeconfig` file using Kubernetes official command line tool ``kubectl``.
+You register and authorize your {k8s} cluster using a [path]``kubeconfig`` file with the {k8s} command line tool ``kubectl``.
```

Comment:
Using `kubectl` is just one way of obtaining the kubeconfig file. For example, in CaaSP, this file can be downloaded from the Velum UI. IMO, how a user obtains this file is out of our scope.